pkg/service/model_build_load_balancer.go

```diff
@@ -26,12 +26,16 @@ import (
 )
 
 const (
-	lbAttrsAccessLogsS3Enabled           = "access_logs.s3.enabled"
-	lbAttrsAccessLogsS3Bucket            = "access_logs.s3.bucket"
-	lbAttrsAccessLogsS3Prefix            = "access_logs.s3.prefix"
-	lbAttrsLoadBalancingCrossZoneEnabled = "load_balancing.cross_zone.enabled"
-	resourceIDLoadBalancer               = "LoadBalancer"
-	minimalAvailableIPAddressCount       = int64(8)
+	lbAttrsAccessLogsS3Enabled                 = "access_logs.s3.enabled"
+	lbAttrsAccessLogsS3Bucket                  = "access_logs.s3.bucket"
+	lbAttrsAccessLogsS3Prefix                  = "access_logs.s3.prefix"
+	lbAttrsLoadBalancingCrossZoneEnabled       = "load_balancing.cross_zone.enabled"
+	lbAttrsLoadBalancingDnsClientRoutingPolicy = "dns_record.client_routing_policy"
+	availabilityZoneAffinity                   = "availability_zone_affinity"
+	partialAvailabilityZoneAffinity            = "partial_availability_zone_affinity"
+	anyAvailabilityZone                        = "any_availability_zone"
+	resourceIDLoadBalancer                     = "LoadBalancer"
+	minimalAvailableIPAddressCount             = int64(8)
 )
```

Comment:
Can we add some test coverage on these attributes? https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/pkg/service/model_build_load_balancer_test.go#L28

Reply:
These are just constants for the possible values for attribute `lbAttrsLoadBalancingDnsClientRoutingPolicy`, for which I already added as tests in https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3470/files#diff-5c328b7b0a8428bb53f5a7b561091d3df4bf46a76538ea0a1590611b584d1196R118.  
```
	partialAvailabilityZoneAffinity          = "partial_availability_zone_affinity"
	anyAvailabilityZone                      = "any_availability_zone"
	resourceIDLoadBalancer                   = "LoadBalancer"
```

Sorry, but I don't get it what you expect me to do.